org.scala-ide.sdt.core/src/org/scalaide/core/internal/extensions/ExtensionCompiler.scala

```diff
@@ -359,3 +381,38 @@ object ExtensionCompiler extends AnyRef with HasLogger {
   }
 
 }
+
+/**
+ * Generator for the creator classes of the IDE extensions. Do not forget to
+ * specify the `saveLocation` before you run this App. It needs to point to
+ * something like `/path/to/scala-ide/org.scala-ide.sdt.core/src`. All the
+ * generated sources are written together with their package hierarchy into this
+ * directory.
+ */
+object ExtensionGenerator extends App {
+
+  private val saveLocation = "/home/antoras/dev/scala/scala-ide/org.scala-ide.sdt.core/src"
```

Comment:
ok, I try it on my local too

2016-02-24 11:03 GMT+01:00 Simon Schäfer <notifications@github.com>:

> In
> org.scala-ide.sdt.core/src/org/scalaide/core/internal/extensions/ExtensionCompiler.scala
> <https://github.com/scala-ide/scala-ide/pull/1071#discussion_r53916473>:
>
> > @@ -359,3 +381,38 @@ object ExtensionCompiler extends AnyRef with HasLogger {
> >    }
> >
> >  }
> > +
> > +/**
> > + * Generator for the creator classes of the IDE extensions. Do not forget to
> > + * specify the `saveLocation` before you run this App. It needs to point to
> > + * something like `/path/to/scala-ide/org.scala-ide.sdt.core/src`. All the
> > + * generated sources are written together with their package hierarchy into this
> > + * directory.
> > + */
> > +object ExtensionGenerator extends App {
> > +
> > +  private val saveLocation = "/home/antoras/dev/scala/scala-ide/org.scala-ide.sdt.core/src"
>
> I ran it in Eclipse, which figured out the classpath by itself. Since this
> app is part of the sdt.core project it shares the same classpath.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/scala-ide/scala-ide/pull/1071/files#r53916473>.
>